### ISSUE TITLE
Help: can you bulk edit identification attributes?

### ISSUE THREAD

**Issue by happiah-madson** [ArctosDB/arctos#7993]
I know that you can bulk add identifications to a group of records, but I want to bulk add an identification attribute to a bunch of records. Specifically, we have a group of specimens that have an identification "nature of identification" attribute that is features. I want to add a "nature of identification" attribute of "molecular" to a bunch of things now that we also have sequence data for them. The identification itself is not changing, so I feel weird about putting a new identification on the record, but additional information about the identification is now available.

@falco-rk

**Comment by mkoo** [comment]
HI Hannah, I dont think we have that specific request but if you send a file of your desired additions in this format (https://arctos.database.museum/loaders/BulkloadIdentification.cfm?action=ld) either Dusty can do a one-time insert or can allow such an bulk change. Basically you would need something like: GUID, SciName, and all the related attributes fields

**Comment by dustymc** [comment]
Yes a tool is possible, but it (or a SQL update) would need to use identificationID (a GUID can carry many identifications). That would need some detail (what precisely can the tool **do**, who can use it, etc.) to become actionable.

Integrating this with a 'manage' option would provide an easy way to get identificationID, and maybe filter out unwanted identifications, provide you a useful view of attributes, etc. (I think that's two requests, but definitely linked.)

----------------------

Wearing my not-developer hat:

>The identification itself is not changing,

Sure it is! Species are hypotheses, "looks like a rat" (features) and "has DNA like a rat" (molecular) are entirely different things. Both of them converging on the same hypothesis/concept adds confidence to the record's rat-ness and suggests that 'rat' as a species concept might be a reasonable thing. Adding an identification carries far more information (and provides proper attribution and a place to hang citations and etc.), than merging multiple IDs into one can.

**Comment by campmlc** [comment]
Using the nature of ID attribute allows the same info to appear in a single identification, e.g. this ID was supported by multiple sources. But if a new identification is added with nature of ID = molecular, then the separate identification can carry metadata with dates and links authors and publications. Our collections tend to use the latter approach, although I can understand scenarios when the first approach is preferable. It would be good to allow both options.

**Comment by dustymc** [comment]
Closing as stale
